### PR TITLE
Update GNPSExport.cpp

### PR DESCRIPTION
**Pull request by eeko-kon**
I added a few more details on the proposed workflow (command examples) mainly - I think it would be great to add a link to an emptyfile.idXML at the IDMapper Requirements section 
"Even in untargeted metabolomics/proteomics, an empty idXML or mzid (peptide annotation format) file is needed as an input which can be found here link/to/emptyfile.idXML.

# Description

Please include a summary of the change and which issue is fixed.

# Checklist:
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

# How can I get additional information on failed tests during CI:
If your PR is failing you can check out 
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. If you click in the column that lists the failed tests you will get detailed error messages.

# Note:
- Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

**Comment by timosachsenberg** [comment]
rebuild jenkins

File: src/topp/GNPSExport.cpp [hunk 1]
```diff
@@ -67,12 +67,12 @@ Nat Methods 17, 905–908 (2020). https://doi.org/10.1038/s41592-020-0933-6
...
 See the FBMN workflow documentation here (https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking/)
 
-In brief, after running an OpenMS "metabolomics" pipeline, the GNPSExport TOPP tool can be used
+In brief, after running an OpenMS "metabolomics" pipeline, the GNPSExport, together with the TextExporter TOPP tool, can be used
 on the consensusXML file and corresponding mzML files to generate the files needed for FBMN on GNPS.
 These two files are:
 
 	- The MS/MS spectral data file (.MGF format) which is generated  with the GNPSExport util.
-	- The feature quantification table (.CSV format) which is generated with the TextExport util.
+	- The feature quantification table (.TXT format) which is generated with the TextExport util.
```
[the remark this answers is not in the document]
Comment (eeko-kon) [reply src/topp/GNPSExport.cpp:75]: I literally just saw this comment- sorry! So I have used those txt files generated by GNPSexport (OpenMS) in FBMN-GNPS and it works perfectly - examples: 
https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=441a29dc057747f094330148d40493e0
https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=af126b5cd46840b79acf4b58cece09ec#

File: src/topp/GNPSExport.cpp [hunk 2]
```diff
@@ -113,14 +110,21 @@ or consensusElement is defined at the IDMapper tool step.
...
-  10. Run the @ref TOPP_TextExporter on the "filtered consensusXML file" to export an .TXT file.
-  11. Upload your files to GNPS and run the Feature-Based Molecular Networking workflow. Instructions are here:
+  	FeatureLinkerUnlabeledKD -in IDMapper_inputFile0.featureXML IDMapper_inputFile1.featureXML -out FeatureLinkerUnlabeledKD.consensusXML
+  7. Run the @ref TOPP_FileFilter on the consensusXML file to keep only consensusElements with at least MS/MS scan (peptide identification). 
+  	FileFilter -id:remove_unannotated_features -in FeatureLinkerUnlabeledKD.consensusXML -out FileFilter.consensusXML
+  8. Run the @ref TOPP_GNPSExport on the "filtered consensusXML file" to export an .MGF file.
+  	GNPSExport -ini iniFile-GNPSExport.ini -in_cm filtered.consensusXML -in_mzml inputFile0.mzML inputFile1.mzML -out GNPSExport_output.mgf
+  9. Run the @ref TOPP_TextExporter on the "filtered consensusXML file" to export an .TXT file.
+  	TextExporter -in FileFilter.consensusXML -out FeatureQuantificationTable.txt
+  10. Upload your files to GNPS and run the Feature-Based Molecular Networking workflow. Instructions are here:
```
[the remark this answers is not in the document]
Comment (eeko-kon) [reply src/topp/GNPSExport.cpp:127]: That's a great idea. Should I ask Ming during GNPS office hours (tomorrow 6 pm)?

**Comment by timosachsenberg** [comment]
@eeko-kon can you check if this is up-to-date with what you are currently doing? I would give it another quick review so we can merge that.

**Review by timosachsenberg — APPROVED** [review]

**Pull request merged**